projects/demo-playwright/utils/page-objects/input-card-grouped.po.ts

```diff
@@ -0,0 +1,21 @@
+import {Locator} from '@playwright/test';
+
+export class TuiInputCardGroupedPO {
+    readonly cardGroupInput = this.host.locator(
+        `[automation-id="tui-input-card-grouped__card"]`,
+    );
+
+    readonly cleanerIcon = this.host.locator(
+        `[automation-id="tui-input-card-grouped__cleaner"]`,
+    );
+
+    readonly cardExpiryInput = this.host.locator(
+        `[automation-id="tui-input-card-grouped__expire"]`,
+    );
+
+    readonly cardCvcInput = this.host.locator(
```

Comment:
All other PO uses term "textfield".
Let's rename `input` and drop prefix `card`:

```suggestion
    readonly numberTextfield = this.host.locator(
        `[automation-id="tui-input-card-grouped__card"]`,
    );

    readonly cleanerIcon = this.host.locator(
        `[automation-id="tui-input-card-grouped__cleaner"]`,
    );

    readonly expiryTextfield = this.host.locator(
        `[automation-id="tui-input-card-grouped__expire"]`,
    );

    readonly cvcTextfield = this.host.locator(
```

Reply:
Usage example:
```ts
const inputCard = new TuiInputCardPO(example);

await inputCard.expiryTextfield.pressSequentially('123');
await expect(inputCard.expiryTextfield).toHaveValue('123');
```